django/contrib/admin/widgets.py

```diff
@@ -196,7 +197,11 @@ def label_and_url_for_value(self, value):
                     obj._meta.app_label,
                     obj._meta.object_name.lower(),
                 ),
-                args=(obj.pk,),
+                args=(
+                    quote(
+                        obj.pk,
+                    ),
+                ),
```

Comment:
```suggestion
                args=(quote(obj.pk),),
```
I know black probably did this, I think the comma after `obj.pk` is not needed and it made black add all the extra lines